src/sentry/issues/endpoints/organization_group_suspect_flags.py

```diff
@@ -40,19 +53,38 @@ def get(self, request: Request, group: Group) -> Response:
             start = start.replace(minute=(start.minute // 5) * 5, second=0, microsecond=0)
             end = end.replace(minute=(end.minute // 5) * 5, second=0, microsecond=0)
 
-        return Response(
-            {
-                "data": [
-                    {"flag": flag, "score": score, "baseline_percent": baseline_percent}
-                    for flag, score, baseline_percent in get_suspect_flag_scores(
-                        organization_id,
-                        project_id,
-                        start,
-                        end,
-                        environments,
-                        group_id,
-                    )
-                ]
-            },
-            status=200,
-        )
+        response_data: ResponseData = {
+            "data": [
+                {"flag": flag, "score": score, "baseline_percent": baseline_percent}
+                for flag, score, baseline_percent in get_suspect_flag_scores(
+                    organization_id,
+                    project_id,
+                    start,
+                    end,
+                    environments,
+                    group_id,
+                )
+            ]
+        }
+
+        # A sum of all the scores is kept to give us a gross understanding on the range of scores
+        # typically seen.
+        total_score = sum(i["score"] for i in response_data["data"])
+        metrics.distribution("flags.suspect.total_score", total_score)
+
+        # Interesting scores are logged for deeper inspection.
+        if response_data["data"] and response_data["data"][0]["score"] >= 1:
```

Comment:
Why do we only look at the first data item?

Reply:
Refactored to something more sane.